libs/core/message-strip/message-strip.component.ts

```diff
@@ -139,7 +152,7 @@ export class MessageStripComponent implements OnInit, OnChanges, CssClassBuilder
     private _translationResolver = inject(TranslationResolver);
 
     /** @hidden */
-    constructor(public readonly elementRef: ElementRef) {}
+    private _hasProjectedLink = false;
```

Comment:
If you make this a signal, then in ngAfterContentInit you set it to true if there's a link inside, you won't need the getter in line 73. You can move the binding to he host in the decorator.